kentikapi/client_integration_test.go

```diff
@@ -220,6 +221,7 @@ type spySyntheticsServer struct {
 	syntheticspb.UnimplementedSyntheticsAdminServiceServer
 	server *grpc.Server
 	url    string
+	done   chan bool
```

Comment:
Please use `struct{}` type as token sent through channel

Reply:
I don't really understand what you mean by this.

Reply:
Ok, I think I found out what you meant.